manifests/f/Fossil/Fossil/2.20/Fossil.Fossil.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Fossil.Fossil
+PackageVersion: "2.20"
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\fossil.exe
+    PortableCommandAlias: fossil
+  InstallerUrl: https://fossil-scm.org/home/uv/fossil-w64-202302092100.zip
+  InstallerSha256: 4C9B56CD1AA3E3AC13478325B15E2EB36DBCEB3E13BB4F8D94BA055D73139F27
```

The x64 file has been updated.
```suggestion
  InstallerSha256: E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855
```

---

Not sure about you, but I'm getting a 0 bytes downloaded .zip file with the hash you suggested 👀 Seems like an issue with the Url

---

Looks like there's a new version available `2.21` and prev Url is not working

---

The hash 4C9...F27 works and has been manually validated. Where is the hash E3B...855 seen?

---

It was in the previous commit https://github.com/microsoft/winget-pkgs/pull/96974/commits/6d05ecafe89b01216c8d2165bf9afcfd5301c366 when version `2.20` was still around and was the latest one.